src/connector/mssql.rs

```diff
@@ -276,16 +277,19 @@ impl Mssql {
 
 #[async_trait]
 impl Queryable for Mssql {
+    #[tracing::instrument(skip(self, q))]
     async fn query(&self, q: Query<'_>) -> crate::Result<ResultSet> {
         let (sql, params) = visitor::Mssql::build(q)?;
         self.query_raw(&sql, &params[..]).await
     }
 
+    #[tracing::instrument(skip(self, q))]
     async fn execute(&self, q: Query<'_>) -> crate::Result<u64> {
         let (sql, params) = visitor::Mssql::build(q)?;
         self.execute_raw(&sql, &params[..]).await
     }
 
+    #[tracing::instrument(skip(self))]
```

Comment:
I'm concerned sql and params could be large or contain information we don't want to leak, here — maybe worrying too much?

Reply:
I want them to the tracing views. It's important to see the SQL there.

Reply:
Maybe we can skip the params?